3d Space Shooter/Assets/Scripts/Ship_Selector_Toggle.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Ship_Selector_Toggle : MonoBehaviour {
```

Comment:
So as far as I can tell, this is a static script? What I mean by that is it only supports one enemy being present on the screen at a time, that has been hand placed there and then manually added to this script to toggle a selector on or off. That simply isn't going to work with what we are doing. Now I could be wrong, but when I pulled and tested it, those were my results. So as it stands, I could accept this, but it really would never be used. What I would do is try and find something that allows you to not just toggle this selector on and off, but allows you to poll the scene, find an instance of an enemy, and enable his selector. Should be a relatively straight forward fix, but I can't promise anything. Good luck.